lib/veda_idmatrix/version.rb

```diff
@@ -1,3 +1,3 @@
 module VedaIdmatrix
-  VERSION = "0.0.10"
+  VERSION = "0.1.0"
```

Comment:
This is not a breaking change, please follow canonical versioning. i.e. This should be 0.0.11

Reply:
It went to 0.1.0 because some dude name Jean fucked up the last version update
it went from version 0.0.9 to 0.0.1

Reply:
I had to correct this, or then next version would end up with duplicate versions.